.github/workflows/actions.yml

```diff
@@ -107,4 +109,57 @@ jobs:
           python -m pip install nox
       - name: Build docs
         run: |
-          nox --error-on-missing-interpreters --non-interactive --session docs
\ No newline at end of file
+          nox --error-on-missing-interpreters --non-interactive --session docs
+
+  publish:
+    name: Publish to PyPi
+    runs-on: ubuntu-latest
+    needs:
+      - check-code-formatting
+      - check-coding-style
+      - check-static-types
+      - tests
+      - build-docs
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: python -m pip install wheel
+      - name: Set version
+        run: |
+          if [[ "${{ github.ref }}" == "refs/tags/"* ]]
+          then
+            VERSION=$(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,' | sed -e 's/^v//')
+          else
+            VERSION=$(date +%Y%m%d).$(date +%H%M%S)
+          fi
+          echo ${VERSION}
+          sed -i "s/__VERSION__/${VERSION}/g" pdfminer/__init__.py
+      - name: Build package
+        run: python setup.py sdist bdist_wheel
+      - name: Publish package to Test PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          user: __token__
+          password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+          repository_url: https://test.pypi.org/legacy/
+          verbose: true
+      - name: Publish package to PyPi
+        if: startsWith(github.ref, 'refs/tags')
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_API_TOKEN }}
+      - name: Create GitHub release
+        if: startsWith(github.ref, 'refs/tags')
+        uses: actions/create-release@v1
+        id: create_release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
+          body: |
+            Copy this from CHANGELOG.md
```

Comment:
This doesn't look right, or would you manually update this?
Worst-case we could always just link to the changelog from here but it's not ideal.

Reply:
Yeah, I was thinking about updating it manually. But now I did some `sed` magic that should auto copy it from the changelog.